### PR TITLE
chore: update expectations

### PR DESCRIPTION
**Pull request by jrandolf-google**

### CODE REVIEW

File: test/TestExpectations.json [hunk 1]
```diff
@@ -545,6 +545,12 @@
     "parameters": ["webDriverBiDi"],
     "expectations": ["FAIL"]
   },
+  {
+    "testIdPattern": "[defaultbrowsercontext.spec] DefaultBrowserContext page.cookies() should work",
+    "platforms": ["darwin", "linux", "win32"],
+    "parameters": ["webDriverBiDi"],
+    "expectations": ["PASS"]
```
Comment (OrKoN) [inline test/TestExpectations.json:552]: Let's inverse the expectation? Remove the expectation on line519 and mark failing as FAIL?

Comment (OrKoN) [reply test/TestExpectations.json:552]: Or are too many still failing?

Comment (jrandolf) [reply test/TestExpectations.json:552]: Tried it, there are still a lot of failures from Firefox

**Review by OrKoN — APPROVED** [review]

File: test/TestExpectations.json [hunk 2]
```diff
@@ -1755,13 +1797,13 @@
   {
     "testIdPattern": "[cookies.spec] Cookie specs Page.setCookie should not set a cookie on a blank page",
     "platforms": ["darwin", "linux", "win32"],
-    "parameters": ["firefox", "webDriverBiDi"],
+    "parameters": ["chrome", "webDriverBiDi"],
```
Comment (OrKoN) [inline test/TestExpectations.json]: why do we need this expectations?

**Pull request merged**